src/api/spec/cassettes/Packages/existing_requests/see_a_request.yml

```diff
@@ -0,0 +1,1118 @@
+---
```

Comment:
This commit's message is wrong, those files are just cassettes recorded... not factories.